tools/specs/Microsoft.Data.SqlClient.nuspec

```diff
@@ -41,6 +41,25 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
         <dependency id="System.Security.Cryptography.Primitives" version="4.3.0" />
         <dependency id="System.Text.Encodings.Web" version="4.7.2" />
       </group>
+      <group targetFramework="net6.0">
+      <dependency id="Microsoft.Data.SqlClient.SNI.runtime" version="5.0.0-preview3.22165.1" exclude="Compile" />
```

Comment:
5.0.0 ?

Reply:
correct, I just pushed it to suppress a conflict I had on the branch. I will fix it in the next push.